storm/src/main/storm/mesos/MesosSupervisor.java

```diff
@@ -122,10 +125,12 @@ public String getAssignmentId() {
 
   @Override
   public void killedWorker(int port) {
+    LOG.info("killedWorker: removing port " + port + " from the 'assigned port state'");
+    String taskId = _state.get(Integer.toString(port));
```

Comment:
I'm just mimicking the existing style, but yes, I agree that String.valueOf() is more canonical.  Also, this is going away with the rewrite to avoid using LocalState.  Please review our internal change first.  As expressed offline, I thought this review would be trivial, hence why I put this public one up already.